dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/user/AuthenticatedUser.java

```diff
@@ -104,6 +104,9 @@
     @Column(nullable = true)
     private Timestamp lastApiUseTime;   // last API use with user's token
 
+    @Column
+    private String notificationsLanguage;
```

Comment:
Can we try with Locale object instead of String? It would be easier to operate on it later.